fpm/src/fpm.f90

```diff
@@ -453,7 +453,11 @@ subroutine cmd_run(settings,test)
         else
             
             if (exists(executables(i)%s)) then
-                call run(executables(i)%s//" "//settings%args)
+	        if(settings%runner .ne. ' ')then
```

Comment:
Wouldn't a check for the allocation status of `settings%runner` runner be clearer here?
```fortran
if (allocated(settings%runner)) then
  ...
```

Reply:
It is allocated to blank when not used. All the options for a particular subcommand are defined, even if to blank. The executable prefix could be used unconditionally in this case, but I thought it was clearer to have a conditional statement.
I was also considering that instead of runner just being a prefix it could be a template where special strings like %FNAME and %BNAME would be replaced with the full pathname or the basename of the file in the future. This syntax acts as a placeholder for something like the C routines (which might be glibc-specific) wordfree(3c) and wordexp(3c) or such as simpler substitution. There are some commands where the command is not just a prefix; thus commands in GNU/Linux and Unix like `xargs -iXX  something XX.  Was waiting to see if anyone complained about --runner just providing a simple prefix syntax.